Makefile.am

```diff
@@ -291,6 +291,18 @@ api: $(GENERATED_PYTHON_FILES)
 	cd $(srcdir); \
 	PYTHONPATH=$(abspath $(top_srcdir)) $(PYTHON) ./makeapi
 
+.PHONY: black
+black: $(GENERATED_PYTHON_FILES)
+	$(PYTHON) -m black \
+	    --config=$(top_srcdir)/pyproject.toml \
+	    $(top_srcdir)
+
+.PHONY: blacklint
+blacklint: $(GENERATED_PYTHON_FILES)
+	$(PYTHON) -m black \
+	    --config=$(top_srcdir)/pyproject.toml --fast --check \
```

Comment:
no clues in the 'check' mode:
```
would reformat test.py
Oh no! 💥 💔 💥
1 file would be reformatted.
```
I guess not all the contributors run the linting locally. Would it be useful to see the diff ( via `--diff`)?
Of course, it's simpler to run `black` over the 'failed' files.

Reply:
Contributors should use ``make black`` or integrate black in their IDE instead. The ``blacklint`` target is for CI.

Reply:
Yes, for CI. And I would like to see the exact reason of CI failure (in particular, the output of this tool to don't run manually on a local machine to get that info).

Reply:
I could add ``--diff` to the argument list. However I don't think it is useful. The correct solution for black violation is always **run black again**.